contracts/Ops.sol

```diff
@@ -269,11 +185,13 @@ contract Ops is Gelatofied, LibOps, IOps {
             "Ops: _createTask: Sender already started task"
```

Comment:
Does this require still make sense for one-time tasks? Lets say you want to schedule 3 limit orders, all with the same data and same eject price. This require would prevent that from happening. Not sure if this is something criticil, but maybe there are use cases where you want to schedule one time tasks that have the same data multiple times

Reply:
Lets think about that.